src/arrayMethodSort.js

```diff
@@ -5,7 +5,43 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let count = 0;
+    let prev = null;
+    let now = null;
+    let totalCompare = null;
+
+    if (!compareFunction) {
+      const defaultCompare = (a, b) => {
+        if (String(a) > String(b)) {
+          return 1;
+        } else if (String(a) < String(b)) {
+          return -1;
+        } else {
+          return 1;
+        }
```

Comment:
```suggestion
        if (String(a) > String(b)) {
          return 1;
        }
        
        if (String(a) < String(b)) {
          return -1;
        }
        
        return 0;
```

Reply:
If an if block always executes a return statement, the subsequent else block is unnecessary. A return in an else if block following an if block that contains a return can be separated into multiple if blocks

More details here
mate-academy.github.io/style-guides/javascript.html#blocks--no-else-return